.palantir/revapi.yml

```diff
@@ -413,6 +413,9 @@ acceptedBreaks:
     - code: "java.method.addedToInterface"
       new: "method java.lang.String org.apache.iceberg.view.ViewVersion::operation()"
       justification: "Add operation API to view version"
+    - code: "java.method.addedToInterface"
```

Comment:
we generally try to avoid breaking the API. What we typically do in such a case is adding a `default` method that throws an `UnsupportedOperationException`